docs/backends.md

```diff
@@ -705,3 +705,66 @@ type: Opaque
 data:
   password: <path:secret-id#key | base64encode>
 ```
+
+### Kubernetes Secret
+
+Setup the backend with default secret from which to conveniently inject values from.
```

Comment:
I don't love the idea of having to have a "default" secret as an env var. I think just specifying the kubernetes secret name in the manifests as inline or annotation is enough.